packages/core/cairo_alpha/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@openzeppelin/wizard-cairo-alpha",
+  "version": "1.0.0",
```

We probably don't want to publish these packages, especially since they are using a different package name, so we could mark this as private.  If we do want to publish an alpha package, we may want to do that from a branch and handle it case by case (probably with `@openzeppelin/wizard-cairo` with an alpha version number).

```suggestion
  "private": true,
  "version": "1.0.0",
```